reference-guide/actions/create-and-manage-smart-actions/use-a-smart-action-form.md

```diff
@@ -1957,7 +1957,6 @@ Collection.register(CustomerForest, Customer)
 {% endtab %}
 
 {% tab title="Laravel" %}
-{% code title="app/Models/Customer.php" %}
```

Comment:
👀

Reply:
It seems like it is breaking the code coloring …